TWLight/settings/base.py

```diff
@@ -126,6 +126,7 @@ def get_django_faker_languages_intersection(languages):
     "django_countries",
     "rest_framework",
     "rest_framework.authtoken",
+    "rosetta",
```

Comment:
I don't think rosetta really belongs in our app since it can't be used in our production workflows. Translations for version-controlled messages need to happen in translatewiki.